crates/goose/Cargo.toml

```diff
@@ -121,7 +119,7 @@ unbinder = "0.1.7"
 winapi = { version = "0.3", features = ["wincred"] }
 
 [dev-dependencies]
-sacp = "10.0.0-alpha.3"
+sacp = { workspace = true }
```

Comment:
sacp now can be a top level dep as we can use a later version of rmcp compatibly